cmp/form/dropdown/BaseDropdownField.js

```diff
@@ -19,14 +20,21 @@ import {HoistField} from '../HoistField';
 export class BaseDropdownField extends HoistField {
 
     static propTypes = {
+        ...HoistField.propTypes,
+
         /** Text to display when control is empty */
         placeholder: PT.string
     };
 
     static defaultProps = {
-        placeholder: 'Select'
-    }
+        placeholder: 'Select',
+        commitOnChange: true
+    };
 
+    constructor(props) {
+        super(props);
+        throwIf(!props.commitOnChange, 'A commitOnChange value of false not implemented on BaseDropdownField.');
```

Comment:
@gportela85 / @lbwexler - I am very sorry I missed this, but what is going on here with commitOnChange being forced to true for BaseDropdownField??? 

This is a badly breaking change for one of our client apps I was just prepping for release where we have a QueryComboField that allows users to type in a company ticker. The combo provides suggestions from their prior entries, but doesn't requireSelection - in fact often a user will be entering a new ticker as the lookup values suggested are only for convenience and rely on the user having entered them before.

It is important that this field not flush its value until the user takes a "commit" action such as blurring, pressing enter, or selecting a complete value from the suggest list. Now it fires a change event on every keystroke, resulting in a service call that throws an exception due to bogus / partial input. 

This was exactly the anti-pattern we were looking to _prevent_ with the entire commit system, and why we wanted commitOnChange to be a relatively rare opt-in prop. We cannot assume every change to the contents of a  BaseDropdownField superclass field is a selection as these allow partial type-ahead input.

Let me know if I am missing something here - we will need to block a release until we can resolve.